src/java_bytecode/jar_file.cpp

```diff
@@ -96,16 +138,17 @@ std::string jar_filet::get_entry(std::size_t i)
 
```

Comment:
you refer to `assert(i<index.size())`? This does hold, `index` holds at most as many elements as the central directory and the entry `i` is translated vie the `filtered_jart`